storybook/src/stories/Filters/molecules/FilterLayout.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+import { FilterLayout } from 'scorer-ui-kit';
+import { select, boolean } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Filters/molecules',
+  component: FilterLayout,
+  decorators: []
+};
+
+const Container = styled.div``;
+
+export const _FilterLayout = () => {
+  const language = select("Language", { English: 'english', Japanese: "japanese" }, "japanese");
+  const hasPageSettings = boolean('Has Page Settings', true);
+  const sizeOptions = [10, 20, 50, 100];
+  const [pageSize, setPageSize] = useState(20);
+
+  const arrayList = [
+    {
+      icon: 'LayoutGrid',
+      id: 'grid',
+      tooltipText: 'Grid View'
+    },
+    {
+      icon: 'LayoutList',
+      id: 'list',
+      tooltipText: 'List View'
+    }
+  ];
+
+  const onSizePerPageChange = useCallback((size: number) => {
+    setPageSize(size);
+  },[]);
+
+  const handleLayout = useCallback((layout: string) => {
+    
+  }, []);
```

Comment:
For these, could change it so it uses the actions feature of Storybook and outputs some information to the actions tab there so we can see what content can be expected? 

There should be some other examples in Storybook you can reference. Perhaps the Standard Button is an easy one!

Reply:
Done